docs/HowToGuides/FirstPullRequest.md

```diff
@@ -0,0 +1,169 @@
+  Since more people are likely to see and be able to answer your question, the
+  question is likely to get an answer sooner. Another benefit of asking in
+  public is that the answers you receive will be helpful to bystanders too.
+- Bug report/Pull request comments: Prefer asking in the bug report/pull request
+  when the question involves additional context specific to the
+  bug report/pull request.
+
+These are suggestions, not rules. For example, it's okay if you ask a broad
+question in a bug report or a pull request.
+
+When asking for help, prefer giving as much information as possible, while
+highlighting the parts that you think are important.
+
+### I didn't get a response from someone. What should I do?
+
+It's possible that you ask someone a question in a bug report/pull request and
+you don't get a response as quickly as you'd like. Maybe they are juggling
+several tasks and the discussion with you accidentally slipped by. Maybe they
+are on vacation or on leave for some reason. If you don't get a response
+within a week, it's okay to politely ping them using an `@` mention with a
+reminder. If you don't get a response for 2-3 weeks in a row, please ping
+someone else.
+
+## Working on a change
+
+Please see our [Getting Started guide][] to understand how to build the code,
+make changes, run tests and debug issues.
+
+[Getting Started guide]: /docs/HowToGuides/GettingStarted.md
+
+## Submitting a pull request
+
+### Tidying up
+
+Alright! You've implemented a change and would like to submit it.
+Double-check that you've tidied your Git history, such as squashing
+work-in-progress commits, and that your commit messages provide context.
+For example, if a commit fixes a bug, then include a "Fixes SR-NNNNN" with the
+bug number in the commit message.
+
+Next, [format your changes](/docs/HowToGuides/FAQ.md#how-do-i-format-my-changes)
+using `clang-format`.
+
+### Pushing and creating a pull request
+
+Assuming you followed the steps in our [Getting Started guide][], you should now
+be able to push your latest changes to GitHub using `git push`.
+
+Next, [create a pull request][] (PR). Usually, if you navigate to
+https://github.com/apple/swift right after pushing your change, GitHub will
+show a helpful "Compare & Pull Request" button.
+
+![Compare & Pull Request button in GitHub UI](/docs/GitHubCreatePRScreenshot.png)
+
+[create a pull request]: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#creating-the-pull-request
+
+## Asking for code review
+
+If you had an active discussion with someone on how to implement your change,
+you can `@` mention them in the PR description and ask for code review.
+If you directly implemented the change without any guidance from anyone else,
+`@` mention someone from GitHub's suggested reviewers. Please ask someone
+though! We don't want you to get stuck because you were not sure who to ask
+for code review.
```

Comment:
Do you think we should mention tagging someone from the code owners list if they’re not sure who to tag? Sometimes GitHub doesn’t show any suggested reviewers.

Reply:
I don't have a quick yes/no answer here. Let me talk about this with some other folks first.

Thanks for bringing this up though. I didn't realize there were situations where GitHub makes no suggestions at all.

Reply:
No worries. And yeah, sometimes I see no suggested reviewers. I _think_ it happens if no one has recently edited a file that is being changed, but I may be wrong. I couldn't find any information on the criteria GitHub uses to suggest reviews.

Reply:
Updated to suggest talking to a code owner in case GitHub doesn't have any suggestion.